### ISSUE TITLE
Data Availability Statement

### ISSUE THREAD

**Issue by erinmr** [FAIRIslandProject/Tetiaroa-Data-Policy#6]
The policy lacks clarity on whether a Data Availability Statement is required or encouraged. Policies should make clear that Data Accessibility Statements are provided in publications indicating how to access the underlying data or to request legitimate access to closed data. 

Conditions for access should also be made clear in the metadata records of the deposited dataset.  
Policies should require tombstone metadata records be maintained after the data may no longer be available to avoid dead ends (e.g., data is destroyed after a retention period). 

https://github.com/FAIRIslandProject/Tetiaroa-Data-Policy/blob/67dbbe7c121ea3738102035d761230d334b76688/TetiaroaDataPolicy.Rmd#L44
